Sources/Prometheus/MetricTypes/Summary.swift

```diff
@@ -119,15 +123,22 @@ public class Summary<NumType: DoubleRepresentable, Labels: SummaryLabels>: Metri
     /// - Parameters:
     ///     - value: Value to observe
     ///     - labels: Labels to attach to the observed value
-    public func observe(_ value: NumType, _ labels: Labels? = nil) {
+    ///     - done: Completion handler
+    ///     - observedValue: Value written to the summary
+    public func observe(_ value: NumType, _ labels: Labels? = nil, _ done: @escaping (_ observedValue: NumType) -> Void = { _ in }) {
```

Comment:
Don't think we need to give the value back to the user.
```suggestion
    public func observe(_ value: NumType, _ labels: Labels? = nil, _ done: @escaping () -> Void = { }) {
```

Reply:
Let's remove them from `Gauge` and `Counter` too then?